compiler/src/iree/compiler/Dialect/Flow/Conversion/TensorToFlow/test/insert.mlir

```diff
@@ -0,0 +1,43 @@
+// RUN: iree-opt --allow-unregistered-dialect --split-input-file --iree-flow-convert-to-flow %s | FileCheck %s
```

Comment:
nit: It doesn't look like `--allow-unregistered-dialect` should be necessary here. Prefer to keep tests focused on just the flags, passes, ops, etc. that are relevant.

Reply:
I just audited our repo and removed a few other instances of that flag that weren't necessary: https://github.com/iree-org/iree/pull/11535

`insert_slice.mlir` in this folder uses that flag for this test case: https://github.com/iree-org/iree/blob/72242333c11ee9dd7c608979da45b92c3a884d6d/compiler/src/iree/compiler/Dialect/Flow/Conversion/TensorToFlow/test/insert_slice.mlir#L58-L70

(the `test.` ops aren't from any dialect)

Maybe a similar test case would be useful for this new conversion pattern?